components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/request/impl/JITProvisioningPostAuthenticationHandler.java

```diff
@@ -334,6 +338,14 @@ private PostAuthnHandlerFlowStatus handleRequestFlow(HttpServletRequest request,
                         username = sequenceConfig.getAuthenticatedUser().getUserName();
                         isUserCreated = true;
                     }
+                    // Check if the associated local account is locked.
+                    if (isAccountLocked(username, context.getTenantDomain())) {
+                        if (log.isDebugEnabled()) {
+                            log.debug("The account is locked for the user " + username);
```

Comment:
Shall we mention the tenant domain as well? WDYT?

Reply:
Fixed https://github.com/wso2/carbon-identity-framework/pull/3598/commits/44b14a3dc4c0be84122db9c92c09d08ce2daf877